accounts/abi/unpack_test.go

```diff
@@ -462,6 +463,36 @@ func TestUnpack(t *testing.T) {
 			if !reflect.DeepEqual(test.want, out) {
 				t.Errorf("test %d (%v) failed: expected %v, got %v", i, test.def, test.want, out)
 			}
+			// Pack
+			if test.err != "" {
```

Comment:
Move that check between lines 444 and 445, replacing `return` with a `continue`,

Reply:
Hmm I think that will leave some stuff untested right?
I thought about it a bit more and I want to unify the packing and unpacking tests some more.

Reply:
Agreed, my bad.